src/util/bit_util.rs

```diff
@@ -335,8 +336,9 @@ impl BitWriter {
 
 
 /// Maximum byte length for a VLQ encoded integer
+/// MAX_VLQ_BYTE_LEN = 5 for i32, and MAX_VLQ_BYTE_LEN = 10 for i64
 // TODO: why maximum is 5?
```

Comment:
we can remove the TODO.

Reply:
Ok, will do. I am still quite surprised that everyone uses MAX_VLQ_BYTE_LEN = 5. It looks like code was copied from Impala codebase and never changed since, but I might be wrong. I am not sure why there is check for MAX_VLQ_BYTE_LEN at all (in Impala and parquet-cpp).